code-reviews/recipes/javascript-and-typescript.md

```diff
@@ -143,7 +143,7 @@ As an alternative [husky](https://github.com/typicode/husky) can be considered t
 
 ## Code Review Checklist
 
-In addition to the [Code Review checklist](../README.md), you should also look for these additional JavaScript related code review items:
+In addition to the [Code Review Checklist](../process-guidance/reviewer-guidance.md) you should also look for these javascript specific code review items
```

Comment:
The only thing I would change in this PR is the "javascript" used here back to "JavaScript"